io/xmlparser/CMakeLists.txt

```diff
@@ -28,10 +28,16 @@ ROOT_STANDARD_LIBRARY_PACKAGE(XMLParser
     Core
 )
 
+if(MSVC AND NOT builtin_clang)
+  # fixes an issue when building with conda not finding iconv.h
```

Comment:
Consider adding a more explicit description of the issue being solved/worked-around.